scripts/pre_push_hook.py

```diff
@@ -345,6 +347,22 @@ def does_diff_include_js_or_ts_files(files_to_lint):
     return False
 
 
+def does_diff_include_ts_files(files_to_lint):
+    """Returns true if diff includes TypeScript files.
+
+    Args:
+        files_to_lint: list(str). List of files to be linted.
+
+    Returns:
+        bool. Whether the diff contains changes in any TypeScript files.
+    """
+
+    for filename in files_to_lint:
```

Comment:
Already a method `does_diff_include_js_or_ts_files` follows the same convention.. Can you ptal!

Reply:
> Already a method does_diff_include_js_or_ts_files follows the same convention

Yeh I agree, it's incorrect in those place as well, do you think we should fix those as well? [It would be great if you can do that!]

Reply:
done